testsuite/classlibrary/TestPatternProxy.sc

```diff
@@ -366,4 +366,19 @@ TestPatternProxy : UnitTest {
 
 
 	}
+
+	test_pbindef_arg_order {
+		var pairs;
+
+		Pbindef(\test).clear;
+		Pbindef(\test, \a, 1, \b, 2, \c, 3);
+
+		pairs = Pbindef(\test).source.pairs;
+		this.assertEquals(pairs[0,2..], [\a, \b, \c], "Pbindef key order should correspond to code");
```

Comment:
we are testing whether the internal structure of the Pbindef matches the one that is specified by the args of the `new` message. For two cases:

1. a fresh Pbindef
2. a Pbindef that has some arguments already

Thinking of it, we could also test if the transformation between
`Pdef(\x, Pbind(\a, 7, \b, 8))`  to `Pbindef(\x, \a, \7, \b, 8)` is consistent in that manner.